src/App.tsx

```diff
@@ -81,6 +81,10 @@ const useStyles = makeStyles((theme: Theme) => ({
       paddingBottom: theme.spacing(25),
       paddingTop: '96px',
     },
+    '& > .moreScreen': {
```

Comment:
@joshbuchea I added this as a way to override the 200px margin bottom. Hoping this looks ok.